src/metatensor/models/utils/export.py

```diff
@@ -9,46 +9,43 @@
 )
 
 
-def export(model: torch.nn.Module) -> MetatensorAtomisticModel:
-    """Export a trained model to allow it to make predictions.
+# TODO: DELETE OR CHANGE THIS FUNCTION.
+# EXPORT IS NOW PER-ARCHITECTURE
```

Comment:
I already talked with @Luthaf .

The `export` function performs some unit existence checks and warns if units are not set. If we can move these checks directly into metatensor (which makes sense for me), we can delete this function.

Until we move the checks I suggest we keep the function, avoiding duplication in the architectures.